CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Changes since v2.24
 ### Breaking changes
 - The columns owneruserid and modifieruserid have been removed as the audit_log effectively serves the same purpose. They were not consistently used. For the attachments the column has been renamed to userid. This is a breaking change for the API as some of these were exposed, but likely not used or useful. (#2823)
 
+### Additions
+- Bona fide pusher script for pushing bona fide status to Elixir.
```

Comment:
We could link to the ticket here, i.e., add `(#2513)`.

Reply:
Added.